src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule4852classannotations/example3/package-info.java

```diff
@@ -0,0 +1,24 @@
+// non-compiled with javac: Used for Testing purpose.
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
please shorted this comment to ` Licensed to the Software Foundation`
lets keep Inputs compact